wurst/foes/creeps/CreepAggro.wurst

```diff
@@ -17,7 +17,13 @@ class AggroEntry
 
 constant aggroMap = new HashMap<unit, LinkedList<AggroEntry>>()
 
-public function unit.addAggro(unit target, int amount)
+interface AggroMergeFn<T>
```

Comment:
@Frotty can you explain your objection more? I do not see a more elegant way to provide both `add` and `set` function in your system. This is just [merge](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#merge-K-V-java.util.function.BiFunction-)

Reply:
I guess the overuse of closures. On second thought, impl is okay, but more important for lowering aggro and "setting" it i think.

Reply:
I was hoping that since there is constant closure for add/set cases, then the performance impact would be acceptable

Thinking about this more, and about your comment regarding 200 being too low for late game, i wonder if this should actually be yet a third merge function, `max`, which uses max of provided and current values. I would be interested in your opinion.